app/assets/stylesheets/home.css

```diff
@@ -1,4 +0,0 @@
-/*
-  Place all the styles related to the matching controller here.
-  They will automatically be included in application.css.
-*/
```

Comment:
This file should have been removed in the commit it was added, since it's not needed.

Reply:
I removed it in another commit (wrote it down in the description of it), is there any problem with that or should I move it to he commit when was created for a cleaner history?

Reply:
I personally prefer the second approach because that means quality over quantity to me. It's ok if you disagree.